scripts/ecs/appsignals/sample-app/task-definitions/spring-petclinic-api-gateway.json

```diff
@@ -0,0 +1,153 @@
+          "value": "http://localhost:4316/v1/traces"
+        },
+        {
+          "name": "OTEL_PROPAGATORS",
+          "value": "tracecontext,baggage,b3,xray"
+        },
+        {
+          "name": "CONFIG_SERVER_URL",
+          "value": "http://config-server-cluster-name.ecs-petclinic:8888"
+        },
+        {
+          "name": "OTEL_RESOURCE_ATTRIBUTES",
+          "value": "aws.log.group.names=/ecs/pet-clinic-api-gateway,service.name=spring-petclinic-demo-api-gateway"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_EXPORTER_ENDPOINT",
+          "value": "http://localhost:4316/v1/metrics"
+        },
+        {
+          "name": "OTEL_METRICS_EXPORTER",
+          "value": "none"
+        },
+        {
+          "name": "JAVA_TOOL_OPTIONS",
+          "value": " -javaagent:/otel-auto-instrumentation/javaagent.jar"
+        },
+        {
+          "name": "SPRING_PROFILES_ACTIVE",
+          "value": "ecs"
+        },
+        {
+          "name": "DISCOVERY_SERVER_URL",
+          "value": "http://discovery-server-cluster-name.ecs-petclinic:8761/eureka"
+        },
+        {
+          "name": "API_GATEWAY_IP",
+          "value": "api_gateway_ip"
+        }
+      ],
+      "mountPoints": [
+        {
+          "sourceVolume": "opentelemetry-auto-instrumentation",
+          "containerPath": "/otel-auto-instrumentation",
+          "readOnly": false
+        }
+      ],
+      "essential": true,
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/ecs/pet-clinic-api-gateway",
+          "awslogs-create-group": "true",
+          "awslogs-region": "region-name",
+          "awslogs-stream-prefix": "ecs"
+        }
+      },
+      "cpu": 256,
+      "memory": 512
+    },
+    {
+      "name": "init",
+      "image": "adot-java-image",
+      "cpu": 0,
+      "portMappings": [],
+      "essential": false,
+      "command": [
+        "cp",
+        "/javaagent.jar",
+        "/otel-auto-instrumentation/javaagent.jar"
+      ],
+      "environment": [],
+      "mountPoints": [
+        {
+          "sourceVolume": "opentelemetry-auto-instrumentation",
+          "containerPath": "/otel-auto-instrumentation",
+          "readOnly": false
+        }
+      ],
+      "volumesFrom": [],
+      "systemControls": []
+    },
+    {
+      "name": "ecs-cwagent",
+      "image": "public.ecr.aws/cloudwatch-agent/cloudwatch-agent:latest",
+      "cpu": 0,
+      "memory": 128,
+      "portMappings": [],
+      "essential": true,
+      "environment": [
+        {
+          "name": "CW_CONFIG_CONTENT",
+          "value": "{\"agent\": {\"debug\": true}, \"traces\": {\"traces_collected\": {\"application_signals\": {\"enabled\": true}}}, \"logs\": {\"metrics_collected\": {\"application_signals\": {\"enabled\": true}}}}"
```

Comment:
Don't need debug as true in the demo I guess?

Reply:
Updated